app/routes/wow.weekly-price-group-delta.tsx

```
@@ -0,0 +1,1365 @@
+                  {allTimestamps.map((timestamp) => (
+                    <option key={timestamp} value={timestamp}>
+                      {formatTimestamp(timestamp)}
+                    </option>
+                  ))}
+                </select>
+              </div>
+              <div>
+                <label className="block text-sm font-medium mb-1">
+                  Reset Range
+                </label>
+                <button
```

Biome (1.9.4)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)